bin/node/src/commands/bundled.rs

```diff
@@ -122,33 +135,41 @@ impl BundledCommand {
                 batch_interval,
                 tx_prover_url,
                 monitor_interval,
+                ntx_ticker_interval,
+                no_ntb,
             } => {
                 Self::start(
                     rpc_url,
                     data_directory,
+                    no_ntb,
                     batch_prover_url,
                     block_prover_url,
                     tx_prover_url,
                     batch_interval,
                     block_interval,
                     monitor_interval,
+                    ntx_ticker_interval,
                 )
                 .await
             },
         }
     }
 
     #[allow(clippy::too_many_arguments)]
+    #[allow(clippy::too_many_lines)]
     async fn start(
         rpc_url: Url,
         data_directory: PathBuf,
+        no_ntb: bool,
         batch_prover_url: Option<Url>,
         block_prover_url: Option<Url>,
         tx_prover_url: Option<Url>,
         batch_interval: Duration,
         block_interval: Duration,
         monitor_interval: Duration,
+        ntx_ticker_interval: Duration,
```

Comment:
Not for this PR, but I do think we should start thinking about how to group these. For example, `no_ntb`, `tx_prover_url`, and `ntx_ticker_interval` should probably be encapsulated in something like `NtbConfig` struct.

Reply:
I agree. I didn't want to change too much for now but I can open an issue to track that.

Reply:
Yes - that would be great!